crates/precompile-native-currency/src/lib.rs

```diff
@@ -115,6 +119,7 @@ where
     Erc20SupportT: pallet_erc20_support::Config,
     AccountIdOf<Erc20SupportT>: From<H160>,
     BalanceOf<Erc20SupportT>: Into<U256> + TryFrom<U256>,
+    AllowanceOf<Erc20SupportT>: TryFrom<U256> + EvmData,
```

Comment:
```suggestion
    AllowanceOf<Erc20SupportT>: TryFrom<U256>,
```

Doesn't seem like it's needed here, can we drop it?

Reply:
It's needed here - https://github.com/humanode-network/humanode/blob/e9d0e8cfadfb50cd099611d55745e195fcd2d266/crates/precompile-native-currency/src/lib.rs#L187.

Reply:
Ah, got it, thanks for the clarification!